flink/v2.0/flink/src/main/java/org/apache/iceberg/flink/sink/DeltaManifestsSerializer.java

```diff
@@ -28,12 +28,12 @@
 import org.apache.iceberg.ManifestFiles;
 import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
 
-class DeltaManifestsSerializer implements SimpleVersionedSerializer<DeltaManifests> {
+public class DeltaManifestsSerializer implements SimpleVersionedSerializer<DeltaManifests> {
```

Comment:
Please mark all of the new public classes to `@Internal`. We use these Flink annotations in the Iceberg connector extensively to limit the external API, and allow us free hand on modifying most of the classes.

Reply:
I missed that one. Thanks for spotting.